components/kyma-metrics-collector/pkg/process/metrics.go

```diff
@@ -48,8 +48,29 @@ var (
 		},
 		[]string{shootNameLabel, instanceIdLabel, runtimeIdLabel, subAccountLabel, globalAccountLabel},
 	)
+	kebTotalClusters = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "clusters_total",
+			Help:      "Number of all clusters got from KEB including trackable and not trackable.",
+		},
+		[]string{"trackable", shootNameLabel, instanceIdLabel, runtimeIdLabel, subAccountLabel, globalAccountLabel},
```

Comment:
We can also define a const for "trackable".

Reply:
done